src/services/rpc.service.js

```diff
@@ -118,6 +118,7 @@ export class FennelRPC {
     } else {
       console.info('message queued: websocket is offline');
       this._offlineOutgoingMessageQueue.unshift(message);
+      throw('Message queued: websocket is offline.')
```

Comment:
We may eventually want to throw an error here, but as of now, this may cause more trouble than its worth right now. Without the error, the service will queue the messages without displaying any issue to the end user and, when the websocket connection is reestablished, all those messages will get drained through to the websocket.

If there is a web socket connection issue, that is something we should be trying to resolve without the end-user knowing about it. Unless it is an internet connection issue on their end, which is an edge case that I doubt we want to worry about right now.

What do you think? I am actually thinking about letting this merge into master and play around with it to see how it plays.